core/src/test/scala/unit/kafka/server/AutoTopicCreationManagerTest.scala

```diff
@@ -263,7 +263,7 @@ class AutoTopicCreationManagerTest {
                                          topicName: String,
                                          isInternal: Boolean): Unit = {
     val topicResponses = autoTopicCreationManager.createTopics(
-      Set(topicName), UnboundedControllerMutationQuota)
+      Set(topicName), UnboundedControllerMutationQuota, None)
```

Comment:
Do we have any test cases which provide a metadata context?

Reply:
Added